x-pack/plugins/monitoring/public/components/no_data/reasons/__tests__/__snapshots__/we_tried.test.js.snap

```diff
@@ -28,7 +28,7 @@ Array [
       No Monitoring data could be found for the selected time period, but we could not find the cluster setting that makes the data unavailable.
     </p>
     <p>
-      There may be data available for a different time period than we have selected. Try adjusting the time filter controls to a time range where the Monitoring data is expected.
+      There may be data available for a different time period than we have selected. Make sure the time filter is set to "Last 1 hour."
     </p>
     <p>
       We are refreshing the search for data in the background. If cluster data is found, we will redirect to the cluster overview page.
```

Comment:
The second half of the first sentence is hard to parse.  Is it necessary to include?  The most important part of the message is the CTA for the user to change the time period.  Can we shorten this message and get the action closer to to top?  Here is a suggestion:

**We couldn't activate monitoring**
--------- 

No monitoring data found. Try setting the time filter to "Last 1 hour" to see if data is available for a different time period.  

If data is in your cluster, you'll see your monitoring dashboards here.

Reply:
>  Try setting the time filter to "Last 1 hour" to see if data is available for a different time period.

If we change `to see if` to "`or see if`, I think it is still clear and concise, but covers more area to troubleshoot.

Do we want to drop "Here might be some things to check"?

![image](https://user-images.githubusercontent.com/908371/41443344-1bbe4fc2-6ff1-11e8-92e4-49c277ae91a4.png)